util/oceantestutil_ve.py

```diff
@@ -0,0 +1,42 @@
+import random
+
+import brownie
+from brownie.convert.main import to_uint
```

Comment:
`to_uint` isn't being used, not sure why imported. In this file and other files.